files/rollup.config.mjs

```diff
@@ -60,7 +63,10 @@ export default {
     addon.gjs(),<% if (typescript) { %>
 
     // Emit .d.ts declaration files
-    addon.declarations('declarations'),<% } %>
+    addon.declarations(
+      'declarations',
+      `<%= packageManager %> exec glint --declaration --project ${configs.ts}`,
```

Comment:
customizing the command here requires addon-dev 8.1+ (which isn't marked latest for some reason?)